src/main/java/com/carina/methods/demoblaze/components/Header.java

```diff
@@ -0,0 +1,66 @@
+package com.carina.methods.demoblaze.components;
+
+import com.carina.methods.demoblaze.pages.AboutUsVideoPage;
+import com.carina.methods.demoblaze.pages.CartPage;
+import com.carina.methods.demoblaze.pages.HomePage;
+import com.carina.methods.demoblaze.pages.LoginMenuPage;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class Header extends AbstractUIObject {
+
+    @FindBy(xpath = "//a[@id='nava']")
+    protected ExtendedWebElement storeTitle;
+
+    @FindBy(xpath = "//a[contains(text(), 'Home')]")
+    private ExtendedWebElement homeHeaderOption;
+
+    @FindBy(xpath = ".//a[@id='login2']")
+    protected ExtendedWebElement loginHeaderOption;
+
+    @FindBy(xpath = ".//a [contains(text(), 'About us')]")
+    protected ExtendedWebElement aboutUsHeaderOption;
+
+
+    @FindBy(xpath = ".//a [@id= 'cartur']")
+    protected ExtendedWebElement cartHeaderOption;
+
+    @FindBy(xpath = ".//a[@id='nameofuser']")
+    protected ExtendedWebElement headerUserGreeting;
+
+    public Header(WebDriver driver, SearchContext searchContext) {
+        super(driver, searchContext);
+    }
+
+
+    public LoginMenuPage clickSignIn() {
```

Comment:
specify Button or Link in the method name
```suggestion
    public LoginMenuPage clickSignInButton() {
```